Kajabi/sage-lib
# SAGE-375: new icons: unlock, move-left, move-right, kajabi-filled, feedback

## Description

Add five new icons: feedback, move-left, move-right, kajabi-filled, and unlock.

## TODO

Add icon files to CDN 
[sage-v2.4.zip](https://github.com/Kajabi/sage-lib/files/8405292/sage-v2.4.zip)


## Screenshots - new icons

| `unlock` | `move-left` | `move-right` | `kajabi-filled` | `feedback` |
| ----------- | ----------- | ----------- | ----------- | ----------- |
|![Screen Shot 2022-04-06 at 10 32 51 AM](https://user-images.githubusercontent.com/24237393/162012235-e1d5e665-865f-48b8-b078-6d45c153b13d.png) | ![Screen Shot 2022-04-06 at 10 29 24 AM](https://user-images.githubusercontent.com/24237393/162011546-be586a41-40db-41ab-9f48-882a6d389c60.png) | ![Screen Shot 2022-04-06 at 10 30 46 AM](https://user-images.githubusercontent.com/24237393/162011809-6e039cbe-2716-44d0-808f-22194a40cab7.png) | ![Screen Shot 2022-04-06 at 10 31 12 AM](https://user-images.githubusercontent.com/24237393/162011894-b43827f4-47ae-4fb7-8406-ea88c651e7e7.png) | ![Screen Shot 2022-04-06 at 10 31 37 AM](https://user-images.githubusercontent.com/24237393/162011988-60066fd6-4b75-4ba1-a8cb-1fc9af3d9418.png) |

## Testing in `sage-lib`

1. Navigate to http://localhost:4000/pages/foundations/icon
2. Ensure the above icons are rendering 


## Testing in `kajabi-products`

1. (**LOW**) New icons added. No impact on app.


## Related
[SAGE-375](https://kajabi.atlassian.net/browse/SAGE-375)
[SAGE-419](https://kajabi.atlassian.net/browse/SAGE-419)
[SAGE-420](https://kajabi.atlassian.net/browse/SAGE-420)
